Documentation/dev/noobaa.md

```diff
@@ -0,0 +1,133 @@
+# Installing NooBaa
+
+Follow the instructions on https://github.com/noobaa/noobaa-operator/ for installing NooBaa.
+
+If you have an existing noobaa install, the management UI and S3 connection information can be found by running `noobaa status`.
+
+It is highly recommended that before beginning you familiarize yourself with NooBaa.
+If you have not already, you should: configure a storage pool using the NooBaa management UI, configure your Tier 1 data placement, and create a bucket for usage by Metering.
+
+## Configuring Metering for NooBaa
+
+Currently noobaa supports TLS, but produces a self-signed cert that we cannot trust due to it not being signed for any of the hostnames we connect to it using.
+You have a few choices:
+
+- Do not use TLS to connect to Noobaa (use the http:// protocol and port 80 instead of https:// with 443).
+- Configure a CA for NooBaa and ensure that `s3.noobaa.svc` is in the subject alternate names of the server certificate.
+  - There is an [open issue](https://github.com/noobaa/noobaa-operator/issues/43#issue-487679497) requesting that certificate generation is handled automatically on Openshift, and contains instructions on how you can create a certificate in Openshift.
+
+### Create the namespace
+
+Before we can start, we need to create the namespace so that we can create the secret containing our NooBaa credentials:
+
+```
+kubectl create ns $METERING_NAMESPACE
+```
+
+### Create noobaa credentials secret
+
+Run the following command to print our your credentials:
+
+```
+noobaa status 2>&1 | grep AWS_
```

Comment:
@chancez alternatively, we can get a more parsible output with `oc get secret noobaa-admin -n noobaa -o yaml`